workers/unity/Packages/com.improbable.gdk.core/GameObjectRepresentation/Delegates.cs

```diff
@@ -0,0 +1,10 @@
+﻿using Improbable.Worker;
+
+namespace Improbable.Gdk.Core.MonoBehaviours
+{
+    // TODO reviewers: shall we put these delegates into their own class/namespace?
```

Comment:
Is there any particular reason they shouldn't be in the same namespace as the other GameObject/MonoBehaviour things just like that? I guess we could have some .Delegates namespace but it looks fine to me as it is too. (Having a class just to keep them in seems weird.)

Reply:
Oversight